packages/common/src/downloads/_internal/file-url-fetchers/fetchExportImageDownloadFile.ts

```diff
@@ -0,0 +1,109 @@
+import { request } from "@esri/arcgis-rest-request";
+import {
+  DownloadOperationStatus,
+  IFetchDownloadFileOptions,
+  IFetchDownloadFileResponse,
+  ServiceDownloadFormat,
+} from "../../types";
+import HubError from "../../../HubError";
+import { getProp } from "../../../objects/get-prop";
+import { IHubEditableContent } from "../../../core/types/IHubEditableContent";
+import { ExportImageFormat } from "../_types";
+
+/**
+ * @private
+ *
+ * Fetches a download file url from an Image Service via the exportImage endpoint.
+ *
+ * @param options options for refining / filtering the resulting download file
+ * @returns a blob containing the download file
+ */
+export async function fetchExportImageDownloadFile(
+  options: IFetchDownloadFileOptions
+): Promise<IFetchDownloadFileResponse> {
+  validateOptions(options);
+
+  const { entity, format, context, progressCallback } = options;
+  progressCallback && progressCallback(DownloadOperationStatus.PENDING);
+
+  const extent = getExportImageExtent(options);
+  const { xmin, xmax, ymin, ymax } = extent;
+  // TODO: do we need to handle latestWkid as well?
+  const { wkid } = extent.spatialReference;
+
+  const requestOptions = { ...context.requestOptions };
+  requestOptions.httpMethod = "GET";
+  requestOptions.params = {
+    bbox: `${xmin},${ymin},${xmax},${ymax}`,
+    bboxSR: `${wkid}`,
+    f: "image",
+    format,
+    mosaicRule:
+      '{"ascending":true,"mosaicMethod":"esriMosaicNorthwest","mosaicOperation":"MT_FIRST"}',
+  };
+
+  // TODO: Figure out whether we want to leverage the server's maxImageWidth and maxImageHeight.
```

Comment:
highlighting for visibility

Reply:
I wonder if we still keep the aspect (square) that we have with the default 400px x 400px, but base it off of the minimum maxImageWidth or maxImageHeight:

```js
const shortestSide = maxImageWidth < maxImageHeight ? maxImageWidth : madImageHeight;
const size = `${shortestSide},${shortestSide}`;
```

Reply:
We could even set define our own max pixel dimensions, but the default 400px is very low resolution for many needs I would assume.

Reply:
I think the real question here is what the use of the "Download Image" function is... b/c the formats we are supporting are "pictures", vs "raster data" that can be used in an image processing system. For "pictures" 400x400 is fine, but that's pretty useless for image processing.

Reply:
I assume there is some value in having the ability to export an image (add an image to a word document, etc) but relatively low compared to how imagery is used.  TIFF's are nice because they are georeferenced and can be dropped onto a map.  Advanced users will likely just want to pull the service into their desktop (qgis, arcgis pro) project directly to consume / manipulate the imagery pixels.  Having the ability to export the underlying tiles ([/exportTiles](https://developers.arcgis.com/rest/services-reference/enterprise/export-tiles-image-service/)) as a tile package (.tpk) is nice for getting the raw data (offline use, etc), but I wonder how many users want this compared to just consuming the service directly.

Reply:
Great thoughts! I have an issue logged for this that I'll add that thought to: https://devtopia.esri.com/dc/hub/issues/10902